README.md

```diff
@@ -64,7 +64,7 @@ yay -S railwayapp-cli
 
 ### Docker
 
-Before using CLI in non-interactive environment make sure that you have created an access token (account or project scoped) and made it available via the `RAILWAY_TOKEN` environment variable.
+Before using the CLI in a non-interactive environment, ensure you have created an access token (account or project-scoped) and set it as the `RAILWAY_TOKEN` environment variable. CI environments are automatically detected by the presence of `CI=true` variable. In these environments, only build logs will be streamed, and the CLI will exit with an appropriate code indicating success or failure.
```

Comment:
Account and Project tokens are named differently -

`RAILWAY_TOKEN` & `RAILWAY_API_TOKEN`

https://github.com/railwayapp/cli/blob/master/src/config.rs#L111
https://github.com/railwayapp/cli/blob/master/src/config.rs#L115

Reply:
Updated readme to only mention project-tokens as we can't really use account-scoped tokens rn (missing project id variable support)

Reply:
Okay, that sounds good, we can tackle that at a later date.